### ISSUE TITLE
[FR] iPad version of the app

### ISSUE THREAD

**Issue by natedwhite** [AppFlowy-IO/AppFlowy#4250]
### Description

Right now, I noticed that testflight only works as a scaled version of the iOS version. A dedicated iPadOS version would be extremely nice to use.

### Impact

It will enable the iPad version to actually function. Right now, it won't even let me sign in on my iPad, even in the scaled version.

**Comment by LucasXu0** [comment]
I haven't run the app on iPadOS yet. There should be many adjustments needed.

**Comment by han-steve** [comment]
+1 on this. Would this be drastically different from the MacOS layout?

**Comment by Shaolo** [comment]
I can’t speak to Flutter, but enabling basic iPad support in Interface Builder was really easy. Just the iOS version enabled to scale to iPad dimensions would be extremely useful.

**Comment by annieappflowy** [comment]
Hello, dear developers of the wonderful AppFlowy application!

I enjoy using the app very much, but I would like to address one rather unpleasant issue regarding the devices on which the application can be loaded.

The story is as follows: I use the application on both my phone and PC. However, when downloading it on my iPad, it does not work quite correctly and has a small interface, similar to that of a phone. I do not know how the application looks on an Android tablet, but if the same issue is observed there, could you please also open this branch for use?

Thank you very much for reading my message. I wish you success in development and all the best!

Best regards,
RedUserHub

**Comment by haron-IV** [comment]
Hi @appflowy @LucasXu0 is there any plan to make it happen? I switched from Notion to Anytype, and I find AppFlowy really cool, but without tablet version it couldn't be treated as a good app ;/

**Comment by fredpan** [comment]
That would be a strong demand. In my circumstance, I cannot use Mac in my work, but I am be able to carry iPad. So it would be nice to have an iPad version there. In addition, is using flutter to generate an ipad version app super easy?

**Comment by lusmo** [comment]
I completely agree. An iPad app is sorely needed. Without it (with a scaled version) it is unusable on the iPad.
